File: token-service-pro1/src/main/java/com/example/tokenservicepro1/VO/Banking.java
package com.example.tokenservicepro1.VO;

import lombok.*;
import org.springframework.data.mongodb.core.mapping.Document;


@Getter
@Setter
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Banking {

    private String bankId;

    private String services_provided;

    private String counter_no;


}

File: banking-ics/src/main/kotlin/com/example/bankingics/entity/Banking.kt
package com.example.bankingics.entity

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.index.Indexed
import org.springframework.data.mongodb.core.mapping.Document

@Document
class Banking {


    @Id
    public var bankId: String? = null

    public var services_provided: String? = null

    public var counter_no: String? = null
}